### ISSUE TITLE
Last paragraph is deprecated

### ISSUE THREAD

**Issue by jeremygottfried** [learn-co-curriculum/react-updating-state#8]
Props are read-only and cannot be modified. Maybe reword the paragraph to clarify that passing props from parent to child does not modify existing properties-- it creates and renders a new instance of the child component with different properties than the previously rendered instance.

**Comment by maxwellbenton** [comment]
Thank you for providing feedback.

We have updated the last portion of this lesson to better clarify the difference between props and state. 

We apologize for any frustration you might have encountered during this process but thank you for helping us ensure that those who follow in your path will not encounter the same problems that you faced.

If you do not believe that this issue has been addressed, please re-open this issue. 💙
